docs/reference/indices/flush.asciidoc

```diff
@@ -2,11 +2,12 @@
 == Flush
 
 The flush API allows to flush one or more indices through an API. The
-flush process of an index basically frees memory from the index by
-flushing data to the index storage and clearing the internal
-<<index-modules-translog,transaction log>>. By
-default, Elasticsearch uses memory heuristics in order to automatically
-trigger flush operations as required in order to clear memory.
+flush process of an index makes sure that any data that is currently only
+persisted in the <<index-modules-translog,transaction log>> is also perminantly
+persisted in Lucene. This reduces recovery times as that data doesn't need to be
+reindexed from the transaction logs after the Lucene indexed is opened. By
+default, Elasticsearch use heuristics in order to automatically
+trigger flushes as required and is rare for users to call the API directly.
```

Comment:
Let’s end the sentence after “required”, drop “and”, add “It” before “is” and add “need to” between “to” and “call”. “It is rare for users to need to call the API directly.”